enclave.go

```diff
@@ -81,12 +83,21 @@ type Config struct {
 	// is required.
 	FQDN string
 
-	// ExtPort contains the VSOCK-facing TCP port that the Web server should
+	// ExtPort contains the TCP port that the Web server should
 	// listen on, e.g. 443.  This port is not *directly* reachable by the
 	// Internet but the EC2 host's proxy *does* forward Internet traffic to
 	// this port.  This field is required.
 	ExtPort uint16
 
+	// UseVsockForExtPort must be set to true if direct communication
+	// between the host and Web server via VSOCK is desired. The daemon will listen
+	// on the enclave's VSOCK address and the port defined in ExtPort.
+	UseVsockForExtPort bool
+
+	// DisableKeepAlives must be set to true if keep-alive connections
+	// should be disabled for the HTTPS service.
+	DisableKeepAlives bool
```

Comment:
Is this intended to mitigate the connection-pool growth? Does it help with the gvproxy footprint in the current setup?

Reply:
> Is this intended to mitigate the connection-pool growth? 

Yes, it is important that connections are short lived so other clients can be served. After running a pcap on nightly, it seems that chromium uses keep-alives when possible, when using `SimpleURLLoader`.

Reply:
I was asking if you tested the affect of this setting with gvproxy. Seems like it's independent of the vsock idea?

Reply:
> I was asking if you tested the affect of this setting with gvproxy. Seems like it's independent of the vsock idea?

I didn't test the affect of the setting when using gvproxy as the relay for incoming requests, if that's what you're asking. I assume performance would be worse in this case.

For outgoing requests, there would be no effect, as this setting only applies to the HTTPS server.